app/controllers/volunteers_controller.rb

```diff
@@ -1,8 +1,8 @@
 class VolunteersController < ApplicationController
   include SmsBodyHelper
 
-  before_action :set_volunteer, except: %i[index new create datatable stop_impersonating]
-  after_action :verify_authorized, except: %i[stop_impersonating]
+  before_action :set_volunteer, except: %i[index new create datatable stop_impersonating table_state table_state_update]
```

Comment:
what was added here should be removed (you moved to the PreferenceSet controller)

Reply:
Pretty much, this file should have no changes.